cmd/zfs_object_agent/zettacache/src/zettacache.rs

```diff
@@ -40,8 +40,8 @@ lazy_static! {
     pub static ref DEFAULT_SLAB_SIZE: u32 = get_tunable("default_slab_size", 16 * 1024 * 1024);
     static ref DEFAULT_METADATA_SIZE_PCT: f64 = get_tunable("default_metadata_size_pct", 15.0); // Can lower this to test forced eviction.
     static ref MAX_PENDING_CHANGES: usize = get_tunable("max_pending_changes", 50_000); // XXX should be based on RAM usage, ~tens of millions at least
-    static ref CHECKPOINT_INTERVAL: u64 = get_tunable("checkpoint_interval", 60);
-    static ref MERGE_PROGRESS_MESSAGE_INTERVAL_MS: u64 = get_tunable("merge_progress_message_interval", 1000);
+    static ref CHECKPOINT_INTERVAL: Duration = Duration::from_secs(get_tunable("checkpoint_interval", 60));
```

Comment:
For these kind of Duration tunables, I usually added a suffix on the tunable name (but not the variable name) to indicate the units (as you did with merge_progress_message_interval_ms), i.e:
```suggestion
    static ref CHECKPOINT_INTERVAL: Duration = Duration::from_secs(get_tunable("checkpoint_interval_secs", 60));
```

Reply:
done